tensorflow/contrib/tensorrt/convert/convert_graph.cc

```diff
@@ -391,9 +414,18 @@ tensorflow::Status ConvertGraphDefToTensorRT(
   int count = 0;
   float total_num_nodes_in_segments = 0.;
   for (auto s : segments) {
-    total_num_nodes_in_segments += s.size();
+    total_num_nodes_in_segments += s.first.size();
+  }
+  // Cluster may not be available
+  std::map<string, tensorflow::Device*> name_to_device_map;
+  if (cluster) {
+    for (const auto dm : cluster->GetDeviceSet()->devices()) {
+      name_to_device_map[dm->name()] = dm;
```

Comment:
Please add a comment why not using DeviceSet's FindDeviceByName() method for future reference. I'm worrying that using the short name of the device won't work in a distributed environment, since many workers will have same short device names and the device pointer may point to a remote device.

Reply:
I already had the comment but I updated it to be clearer.